### ISSUE TITLE
[BUG] Mediatek device stops beaconning after some time

### ISSUE THREAD

**Issue by rmelotte** [prplfoundation/prplMesh#1010]
The mediatek device used for the certification tests in CI sometimes stops beaconning completely.

When it happens, the error message is usually pretty specific. For example:
```
No Beacon frame found, check CTT Agent1 is operating on the configured channel
```

A reboot fixes it.

It happened twice in 2 days lately.

Example of a job where it happens:
https://gitlab.com/prpl-foundation/prplMesh/-/jobs/483434203

**Comment by rmelotte** [comment]
Same issue here: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/484791794/artifacts/browse/logs/FAIL/MAP-4.8.3_ETH_FH5GL/

In the UCC logs, there is always this (note the `N/A` reply):
```
INFO     MediatekAGT (127.0.0.1:9002) ---> dev_get_parameter,program,map,ruid,0x000000000001,ssid,Multi-AP-1,parameter,macaddr
INFO     MediatekAGT (127.0.0.1:9002) <-- status,COMPLETE,macaddr,N/A
```

**Comment by rmelotte** [comment]
The mediatek sigma agent issues the following command on the device:
```
wappctrl ra0 map get_macaddr Multi-AP-1 000000000001
```

This is supposed to get the MAC of the radio that has the SSID "Multi-AP-1" and the RUID "0x000000000001".
The resulting MAC is supposed to be in `/tmp/map_macaddr.txt` on the device.

If the result is N/A, it could mean that either:
- the RUID is not correct
- the radio does not use the same SSID (which could mean for example that autoconfig failed, and the device is still using the initial "MAP-AP-Unconfig" SSID)

**Comment by rmelotte** [comment]
Fixed with commit `03bcb2050ecda14f850f167cc2b909e7bdabcf74` in the `easymesh_cert` repo: https://git.prpl.dev/prplmesh/wfa-certification/easymesh_cert/-/commit/03bcb2050ecda14f850f167cc2b909e7bdabcf74

The fix for the mediatek device is in commit `113c03ec23f4137c026dc3fbd838abbe1b07cb25`: https://git.prpl.dev/prplmesh/wfa-certification/mediatek_ucc_agent/-/commit/113c03ec23f4137c026dc3fbd838abbe1b07cb25

**Comment by rmelotte** [comment]
Just for the record, here are 2 tests passing successively, the first one being run right after a test were the issue occurred (without any reboot):
- `MAP-5.4.1_ETH_FH24G` with the mediatek device as CTTAgent1: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/498020178

- `MAP-4.8.3_ETH_FH5GL`: https://gitlab.com/prpl-foundation/prplMesh/-/jobs/498025912
